utils/getClaimStatus.tsx

```diff
@@ -101,9 +101,18 @@ export function buildNextSteps(
   const json = scenarioObject[whichSteps]
   for (const [index, value] of json.entries()) {
     const keys = ['scenarios', scenarioString, whichSteps, index.toString()]
-    steps.push(buildTransLineContent(value, buildI18nKey(keys)))
+    if (Array.isArray(value.subBullets)) {
```

Comment:
I haven't quite gotten this working recursively and want to pass this to Kat, so I'll put that change in my next PR. Won't change the functionality.